contracts/src/tests/test_ownable.cairo

```diff
@@ -4,155 +4,10 @@ use starknet::testing::set_caller_address;
 use starknet::testing::set_contract_address;
 use zeroable::Zeroable;
 
-use openzeppelin::access::ownable::ownable::OwnableComponent;
 use openzeppelin::access::ownable::interface::{
     IOwnableTwoStep, IOwnableTwoStepDispatcher, IOwnableTwoStepDispatcherTrait
 };
-use OwnableComponent::InternalTrait;
 
-#[starknet::contract]
-mod OwnableMock {
-    use super::OwnableComponent;
-    use starknet::ContractAddress;
-
-    component!(path: OwnableComponent, storage: ownable, event: OwnableEvent);
-
-    #[abi(embed_v0)]
-    impl OwnableImpl = OwnableComponent::OwnableTwoStepImpl<ContractState>;
-    impl InternalImpl = OwnableComponent::InternalImpl<ContractState>;
-
-    #[storage]
-    struct Storage {
-        #[substorage(v0)]
-        ownable: OwnableComponent::Storage
-    }
-
-    #[event]
-    #[derive(Drop, starknet::Event)]
-    enum Event {
-        #[flat]
-        OwnableEvent: OwnableComponent::Event
-    }
-
-    #[constructor]
-    fn constructor(ref self: ContractState, owner: ContractAddress) {
-        self.ownable.initializer(owner);
-    }
-}
-
-type ComponentState = OwnableComponent::ComponentState<OwnableMock::ContractState>;
-
-fn STATE() -> ComponentState {
-    OwnableComponent::component_state_for_testing()
-}
```

Comment:
This function started failing since I think the method isn't publicly visible outside of the package anymore. We were duplicating OZ tests anyway so I removed them